app/helpers/commonwealth_vlr_engine/catalog_helper_behavior.rb

```diff
@@ -125,7 +125,9 @@ def render_item_breadcrumb(document, link_class = nil)
     # @param files_hash [Hash] output of CommonwealthVlrEngine::Finder.get_files
     # @return [Boolean]
     def render_pdf_viewer?(files_hash)
-      !has_image_files?(files_hash) && has_pdf_files?(files_hash) && !has_playable_audio?(files_hash)
+      (!has_image_files?(files_hash) || files_hash[:image].count < 2) &&
```

Comment:
use `size` instead of `count` since its more performant and is there any way we could make this more readable by making these conditions their own methods.